Lecture-4/Task4/Task4/Application.java

```diff
@@ -0,0 +1,78 @@
+package Task4;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Application {
+
+    private static ArrayList<Runner> listNotSorted;
```

Comment:
Use interface types whenever it's possible: `private static List<Runner> listNotSorted;`